customer-terminate-service/app/src/test/java/se/inera/intyg/cts/application/api/TerminationControllerTest.java

```diff
@@ -0,0 +1,104 @@
+package se.inera.intyg.cts.application.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_CREATED;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_CREATOR_HSA_ID;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_CREATOR_NAME;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_HSA_ID;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_ORGANIZATION_NUMBER;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_PERSON_ID;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_PHONE_NUMBER;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_STATUS;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.DEFAULT_TERMINATION_ID;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+import se.inera.intyg.cts.application.dto.CreateTerminationDTO;
+import se.inera.intyg.cts.application.dto.TerminationDTO;
+import se.inera.intyg.cts.application.service.TerminationService;
+
+@ExtendWith(MockitoExtension.class)
+class TerminationControllerTest {
+
+    @Mock
+    private TerminationService terminationService;
+    @InjectMocks
+    private TerminationController terminationController;
+
+    private final TerminationDTO terminationDTO = new TerminationDTO(DEFAULT_TERMINATION_ID, DEFAULT_CREATED,
```

Comment:
Move creation of TerminationDTO to TerminationTestDataBuilder to create a default dto Reduce the number of places to update if/when changes are made.